### ISSUE TITLE
BRouter Web : search flat route ?

### ISSUE THREAD

**Issue by Clemens-Ratte-Polle** [nrenner/brouter-web#788]
hi.
Can i search for a flat route without big heights?
https://brouter.de/brouter-web/
This would be fine.
Also an altitude profile would be fine to display optionally.
thx

**Comment by Clemens-Ratte-Polle** [comment]
ok, i did not find this option in the drop down in the top left, but hidden in the option :
"consider_elevation
Set to false to ignore elevation in routing"
This option should be easier accessable.

**Comment by EssBee59** [comment]
> This option should be easier accessable.

I agree, the parameter "consider_elevation" is very useful and should be easier accessible!

The parameters are defined in the profile (here the "trekking" profile) , and this is part of "brouter", not "brouter-web" 
(https://github.com/abrensch/brouter/issues/)

This is one of the many weakness I see in this profile, I hope to get a change soon.
Till a change, you can use my own trekking profile with this link:
https://brouter.de/brouter-web/#map=13/49.9945/8.6577/osm-mapnik-german_style&profile=trekking_SB

Regards
EssBee
